po/zh.po

```diff
@@ -328,51 +328,51 @@ msgstr "%s 第 %u 行"
 #: src/configparser.cpp:132
 #, c-format
 msgid "unknown command `%s'"
-msgstr "未知的命令 `%s' "
+msgstr "未知的命令`%s'"
 
 #: src/configparser.cpp:139
 #, c-format
 msgid "Error while processing command `%s' (%s): %s"
-msgstr "当处理命令`%s'(%s)时出错：%s"
+msgstr "当处理命令`%s' (%s)时出错：%s"
 
 #: src/controller.cpp:154 src/pbcontroller.cpp:233
 #, c-format
 msgid "Starting %s %s..."
-msgstr "启动 %s %s..."
+msgstr "启动 %s %s......"
```

Comment:
Double ellipses again. And heaps more down below (I'm not gonna comment on all of them).